osu.Game/Overlays/Chat/Tabs/ChannelTabControl.cs

```diff
@@ -38,7 +38,7 @@ public ChannelTabControl()
                 Margin = new MarginPadding(10),
             });
 
-            AddTabItem(selectorTab = new ChannelSelectorTabItem(new Channel { Name = "+" }));
+            AddTabItem(selectorTab = new ChannelSelectorTabItem(new ChannelSelectorTabChannel()));
```

Comment:
I'd drop the argument and just do this inside the `ChannelSelectorTabItem` class.